tar/chunks/chunks_stats.c

```diff
@@ -414,3 +414,50 @@ chunks_stats_free(CHUNKS_S * C)
 	free(C->cachepath);
 	free(C);
 }
+
+/**
+ * chunks_initialize(const char * cachepath):
+ * Initialize the chunk directory (file) in the ${cachepath}.  Return 0 on
+ * success, -1 on error, and 1 if the file already exists.
+ */
+int
+chunks_initialize(const char * cachepath)
+{
+	char * directory_filename;
+	RWHASHTAB * ht;
+	struct chunkstats stats_extra;
+
+	/* Bail if ${chunkpath}/directory already exists. */
+	if (asprintf(&directory_filename, "%s/directory", cachepath) == -1)
```

Comment:
This might make sense as a `chunks_directory_exists` function?  That's the only file where the name "/directory" appears.